staging/rustfmt.toml

```diff
@@ -1,4 +1,4 @@
-edition = "2018"
+edition = "2021"
```

Comment:
The edition bump ended up in `staging: move rustfmt.toml to workspace directory` was that intended?

Reply:
Oh yeah. I will switch them around